### PR TITLE
verify-bugs: Report consistently

### PR DESCRIPTION
**Pull request by thegreyd**
Wrap validate methods in try-except and have exceptions go to the report at the end.
This is so that even if we break, we always report on previously completed validations.
Move reporting to `report()` in BugValidator class

Test:
`./elliott -g openshift-4.16 --assembly stream verify-bugs --output slack`

### CODE REVIEW

File: elliott/elliottlib/cli/verify_attached_bugs_cli.py [hunk 1]
```diff
@@ -83,32 +85,28 @@ async def verify_attached_bugs(runtime: Runtime, verify_bug_status: bool, adviso
     # placeholder
     non_flaw_bugs = [b for b in bugs if b.is_ocp_bug()]
 
-    validator.validate(non_flaw_bugs, verify_bug_status, no_verify_blocking_bugs, is_attached=True)
+    try:
```
Comment (thegreyd) [inline elliott/elliottlib/cli/verify_attached_bugs_cli.py:88]: move logic in try-catch, no other change

File: elliott/elliottlib/cli/verify_attached_bugs_cli.py [hunk 2]
```diff
@@ -83,32 +84,28 @@ async def verify_attached_bugs(runtime: Runtime, verify_bug_status: bool, adviso
...
-
-    if validator.problems:
-        if validator.output != 'slack':
-            print("Found the following problems, please investigate")
-            for p in validator.problems:
-                print(p)
-        exit(1)
+        if verify_flaws:
+            await validator.verify_attached_flaws(advisory_bug_map)
+    except Exception as e:
```
Comment (vfreex) [inline elliott/elliottlib/cli/verify_attached_bugs_cli.py:104]: Thinking we should exit with code non-zero on error.

File: elliott/elliottlib/cli/verify_attached_bugs_cli.py [hunk 3]
```diff
@@ -153,15 +150,13 @@ async def verify_bugs(runtime, verify_bug_status, output, no_verify_blocking_bug
...
-    await validator.close()
-
-    if validator.problems:
-        if validator.output != 'slack':
-            red_print("Some bug problems were listed above. Please investigate.")
-        exit(1)
+    try:
+        validator.validate(ocp_bugs, verify_bug_status, no_verify_blocking_bugs)
+    except Exception as e:
+        validator._complain(f"Error validating bugs: {e}")
```
Comment (vfreex) [inline elliott/elliottlib/cli/verify_attached_bugs_cli.py:156]: Same here

**Review by vfreex — COMMENTED** [review]
/approve
/lgtm

**Review by vfreex — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**